docs/INSTALLATION.md

```diff
@@ -0,0 +1,272 @@
+* [MariaDB](https://mariadb.org/)
+
+The [SVG App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-svg) requires:
+
+* [Tuoris](https://github.com/fvictor/tuoris) (installation instructions available on GitHub repository)
+
+### Downloading source code
+
+All OVE projects can be downloaded from their GitHub repositories:
+
+* OVE Core: [master](https://github.com/ove/ove) | [releases](https://github.com/ove/ove/releases)
+* OVE Apps: [master](https://github.com/ove/ove-apps) | [releases](https://github.com/ove/ove-apps/releases)
+* OVE Services: [master](https://github.com/ove/ove-services) | [releases](https://github.com/ove/ove-services/releases)
+* OVE Asset Services: [master](https://github.com/ove/ove-asset-services) | [releases](https://github.com/ove/ove-asset-services/releases)
+
+The `master` branch of each repository contains the latest code, and can also be cloned if you intend to contribute code or fix issues:
+
+```sh
+git clone https://github.com/ove/ove
+```
+
+Once the source code has been downloaded OVE can be installed either on a local Node.js environment (such as PM2's Node.js environment) or within a Docker environment. The two approaches are explained below.
+
+### Compiling source code for a local Node.js environment
+
+Once you have cloned or downloaded the code, OVE can be compiled using the [Lerna](https://lernajs.io/) build system:
+
+```sh
+cd ove
+lerna bootstrap --hoist
+lerna run clean
+lerna run build
+lerna run test
+```
+
+Instructions above are only provided for the [**OVE Core**](https://github.com/ove/ove) repository. The steps to follow are similar for other repositories.
+
+#### Starting and stopping PM2 process manager
+
+The SVG app requires an instance of [Tuoris](https://github.com/fvictor/tuoris) to be available before starting it. To start Tuoris run:
+
+```sh
+pm2 start index.js -f -n "tuoris" -- -p PORT -i 1
+```
+
+OVE can then be started within the PM2 environment by running:
+
+```sh
+OVE_HOST="OVE_CORE_HOST:PORT" TUORIS_HOST="TUORIS_HOST:PORT" pm2 start pm2.json
+```
+
+By default, OVE core, and all services run on `localhost`, which should be used in place of `OVE_CORE_HOST` and `TUORIS_HOST` names above. The default `PORT` numbers for OVE core and Tuoris are provided in the [Running OVE](#running-ove) section.
+
+Once the services have started, you can check their status by running:
+
+```sh
+pm2 status
+```
+
+Then, to check logs of all services, run:
+
+```sh
+pm2 logs
+```
+
+To stop the PM2 environment run:
+
+```sh
+pm2 stop pm2.json
+```
+
+To clean-up the PM2 environment run:
+
+```sh
+pm2 delete pm2.json
+```
+
+### Compiling source code for a Docker environment
+
+This approach currently works only for Linux and MacOS environments. The `build.sh` script corresponding to each repository can be found under the top most folder of the cloned or downloaded repository or within a `packages/PACKAGE_NAME` folder corresponding to each package.
```

Comment:
I think that approach is expect to work for all unix-like operating systems, but has been tested only on Linux and MacOS

Reply:
"**most** unix-like operating systems" is vague. Unix-like is much broader than Unix to start with an even in the Unix domain, some functions such as readlink are not available in some Solaris versions. So, (at least personally), I would rather not go there with our docs.